BookReaderTemplate/BookNavigator/util/debouncer.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Wait until some time has passed before executing a callback.
+ *
+ * @param {Function} callback
```

Comment:
Note to self: In a future PR we should dissolve this to use the existing debounce function.